### PR TITLE
Efficientnetv2

### PR DESCRIPTION
**Pull request by yassineAlouini**
In this PR, similarly to the `EfficientNet` PR, I add the `EfficientNetV2` new documentation.

### CODE REVIEW

File: docs/source/models/efficientnetv2.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
+EfficientNetV2
+============
+
+.. currentmodule:: torchvision.models
+
+The EfficientNetV2 model is based on the `EfficientNetV2: Smaller Models and Faster Training <https://arxiv.org/pdf/2104.00298>`__
```
Comment (NicolasHug) [inline docs/source/models/efficientnetv2.rst:6]:
```suggestion
The EfficientNetV2 model is based on the `EfficientNetV2: Smaller Models and Faster Training <https://arxiv.org/abs/2104.00298>`__
```

File: torchvision/models/efficientnet.py [hunk 2]
```diff
@@ -848,8 +848,19 @@ def efficientnet_v2_s(
     `"EfficientNetV2: Smaller Models and Faster Training" <https://arxiv.org/abs/2104.00298>`_.
```
Comment (NicolasHug) [inline torchvision/models/efficientnet.py:848]:
```suggestion
    `EfficientNetV2: Smaller Models and Faster Training <https://arxiv.org/abs/2104.00298>`_.
```

File: torchvision/models/efficientnet.py [hunk 3]
```diff
@@ -874,8 +885,19 @@ def efficientnet_v2_m(
...
```
Comment (NicolasHug) [inline torchvision/models/efficientnet.py:885]:
```suggestion
    `EfficientNetV2: Smaller Models and Faster Training <https://arxiv.org/abs/2104.00298>`_.
```

**Review by NicolasHug — APPROVED** [review]
Thanks @yassineAlouini . Some minor comments (that I will resolve myself), but LGTM. Thanks a lot for your PRs!

**Comment by yassineAlouini** [comment]
Thanks @NicolasHug for catching the few parts that I haven't fixed. :ok_hand:

**Pull request merged**